From 33d4b8611e399d3a4d805030ab69982c5568aba8 Mon Sep 17 00:00:00 2001
From: Olivier Moysan <olivier.moysan@foss.st.com>
Date: Wed, 11 Oct 2023 14:52:15 +0200
Subject: [PATCH 0674/1141] arm64: dts: st: add power domain in sai sub nodes
 on stm32mp251

Add power domain in SAI sub nodes for STM32MP251 device tree.

Change-Id: I492ba534582c1b8478eaf2326ed69db480122c06
Signed-off-by: Olivier Moysan <olivier.moysan@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/336785
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Domain-Review: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
---
 arch/arm64/boot/dts/st/stm32mp251.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

--- a/arch/arm64/boot/dts/st/stm32mp251.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp251.dtsi
@@ -1368,6 +1368,7 @@
 					clocks = <&rcc CK_KER_SAI1>;
 					clock-names = "sai_ck";
 					dmas = <&hpdma 73 0x43 0x21 0>;
+					power-domains = <&CLUSTER_PD>;
 					status = "disabled";
 				};
 
@@ -1378,6 +1379,7 @@
 					clocks = <&rcc CK_KER_SAI1>;
 					clock-names = "sai_ck";
 					dmas = <&hpdma 74 0x43 0x12 0>;
+					power-domains = <&CLUSTER_PD>;
 					status = "disabled";
 				};
 			};
@@ -1403,6 +1405,7 @@
 					clocks = <&rcc CK_KER_SAI2>;
 					clock-names = "sai_ck";
 					dmas = <&hpdma 75 0x43 0x21 0>;
+					power-domains = <&CLUSTER_PD>;
 					status = "disabled";
 				};
 
@@ -1413,6 +1416,7 @@
 					clocks = <&rcc CK_KER_SAI2>;
 					clock-names = "sai_ck";
 					dmas = <&hpdma 76 0x43 0x12 0>;
+					power-domains = <&CLUSTER_PD>;
 					status = "disabled";
 				};
 			};
@@ -1438,6 +1442,7 @@
 					clocks = <&rcc CK_KER_SAI3>;
 					clock-names = "sai_ck";
 					dmas = <&hpdma 77 0x43 0x21 0>;
+					power-domains = <&CLUSTER_PD>;
 					status = "disabled";
 				};
 
@@ -1448,6 +1453,7 @@
 					clocks = <&rcc CK_KER_SAI3>;
 					clock-names = "sai_ck";
 					dmas = <&hpdma 78 0x43 0x12 0>;
+					power-domains = <&CLUSTER_PD>;
 					status = "disabled";
 				};
 			};
@@ -1533,6 +1539,7 @@
 					clocks = <&rcc CK_KER_SAI4>;
 					clock-names = "sai_ck";
 					dmas = <&hpdma 79 0x63 0x21 0>;
+					power-domains = <&CLUSTER_PD>;
 					status = "disabled";
 				};
 
@@ -1543,6 +1550,7 @@
 					clocks = <&rcc CK_KER_SAI4>;
 					clock-names = "sai_ck";
 					dmas = <&hpdma 80 0x43 0x12 0>;
+					power-domains = <&CLUSTER_PD>;
 					status = "disabled";
 				};
 			};
